### PR TITLE
Feature/controller

### PR DESCRIPTION
**Pull request by marceldobrica**
A simple controller

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,52 +1,19 @@
...
-/bin/*
-!bin/console
-!bin/symfony_requirements
+###> symfony/framework-bundle ###
+/.env.local
+/.env.local.php
+/.env.*.local
+/config/secrets/prod/prod.decrypt.private.php
+/public/bundles/
+/var/
```
Comment (calinpristavu) [inline .gitignore:8]: add `/idea/` to gitignore, so you don't commit phpstorm specific files

File: src/Controller/PageController.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class PageController extends AbstractController
+{
+    /**
+     * @Route("/test")
```
Comment (calinpristavu) [inline src/Controller/PageController.php]: Please always specify the method in the route definition

File: src/Controller/PageController.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class PageController extends AbstractController
```
Comment (calinpristavu) [inline src/Controller/PageController.php:10]: try to not extend `AbstractController`, but tag all controllers with `controller.service_arguments` tag

File: config/services.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+    # makes classes in src/ available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    App\:
+        resource: '../src/'
+        exclude:
+            - '../src/DependencyInjection/'
+            - '../src/Entity/'
+            - '../src/Kernel.php'
+
```
Comment (calinpristavu) [inline config/services.yaml:22]: this is where you should add your tag to all the controllers in the `src/controller` dir

File: .gitignore (hunk 1 above)
Comment (marceldobrica) [reply .gitignore:8]: Solved

File: src/Controller/PageController.php (hunk 3 above)
Comment (marceldobrica) [reply src/Controller/PageController.php:10]: ok.

**Comment by comanpatricia** [comment]
All good!

**Pull request merged**